gen/kneeboard.py

```diff
@@ -293,7 +322,7 @@ def write(self, path: Path) -> None:
                 [comm.name, "", "", "", self.format_frequency(comm.freq)]
             )
 
-        for a in self.awacs:
+        for a in unique_awacs:
             comm_ladder.append(
                 [a.callsign, "AWACS", "", "", self.format_frequency(a.freq)]
             )
```

Comment:
Just remove this. We don't need to duplicate this info. The kneeboard is already pretty cramped.

We might want to have a new page for the full comm ladder at some point, but that's out of scope for this change IMO.